config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,25 @@
+# Be sure to restart your server when you modify this file.
+
```

Comment:
Unused?

Reply:
Rails added this file too.

I should mention, there's an "rake rails update" task that adds these files. If you made a new 5.2 project it would include them.